src/main/java/org/eclipsefoundation/git/eca/service/impl/PagintationProjectsService.java

```diff
@@ -1,12 +1,13 @@
-/*******************************************************************************
- * Copyright (C) 2020 Eclipse Foundation
- * 
- * This program and the accompanying materials are made
- * available under the terms of the Eclipse Public License 2.0
- * which is available at https://www.eclipse.org/legal/epl-2.0/
- * 
- * SPDX-License-Identifier: EPL-2.0
- ******************************************************************************/
+/**
+ * ***************************************************************************** Copyright (C) 2020
+ * Eclipse Foundation
+ *
+ * <p>This program and the accompanying materials are made available under the terms of the Eclipse
```

Comment:
Are these p tags intentional?

Reply:
p tags are fine in javadocs, they explicitly create paragraphs and tend to output cleaner. Probably put there by the formatter

Reply:
issue is that this is not javadoc here. It's a copyright header. I'm pretty sure there is a rule in the Eclipse formatter  engine (independently of the formatting style you're using) to exclude such headers from being formatted.